registry-library/library/library.go

```diff
@@ -55,16 +55,22 @@ var (
 	DevfileAllMediaTypesList = []string{DevfileMediaType, DevfilePNGLogoMediaType, DevfileSVGLogoMediaType, DevfileVSXMediaType, DevfileArchiveMediaType}
 )
 
-// GetRegistryStacks returns the list of stacks, more specifically
-// it gets the content of the index (index.json) of the specified registry
-// for listing the stacks
-func GetRegistryStacks(registry string) ([]indexSchema.Schema, error) {
+// GetRegistry returns the list of stacks and/or samples, more specifically
+// it gets the stacks and/or samples content of the index of the specified registry
+// for listing the stacks and/or samples
+func GetRegistry(registry string, devfileType indexSchema.DevfileType) ([]indexSchema.Schema, error) {
```

Comment:
Personally, I don't like the name `GetRegistry` for this function. I understand why it was renamed, but personally I think it's too ambiguous.

`GetRegistryIndex` maybe? Any other ideas?

Reply:
We almost need a name for something that can refer to either stacks _or_ samples, but I don't know what that would be 🤔

Reply:
I think `GetRegistryIndex` is more specific and it looks good to me.

Reply:
Updated